README.md

```diff
@@ -55,383 +51,248 @@ of the [Federated Identity Community Group](https://fedidcg.github.io/).
-        configURL : "https://example.com/FEDCM.json",
-      },
-      {
-        origin : "https://login.idp.net", // Actually fine!
-      },
-      {
-        loginURL : "https://auth.example.biz/login" // Invalid combination, can't have loginURL and other providers
-      },
-      {
-        configURL : "https://example.com/FEDCM.json", // This provider entry will never be valid,
-        loginURL : "https://auth.example.biz/login"   // even if it is the only one in the list.
-      },
-    ]
-  }
-})
-```
+#### Distinguishing between `sign-up` and `sign-in` without an `accounts_endpoint`
 
-## Relying Party API, Using a Credential
+FedCM, by default, makes a distinction between sign-up and sign-in via a property called approved_clients in the user profile information that is received in the [fetch the accounts] step: [https://w3c-fedid.github.io/FedCM/#fetch-the-accounts](): if the clientId passed in the JS call is not a member of approved_clients, it means that this client was never approved previously by the user.
 
-The RP can use the Credential as an object once it is obtained, as it would with FedCM. This can be used to verify that the user has selected an account with a given IdP, providing an `origin` field on the credential by analogy to the `configUrl` from the [multi IdP proposal](https://github.com/w3c-fedid/multi-idp). It also provides access to a token from the IDP, provided that the `tokenURL` parameter was provided when the credential was stored.
+This could also be accomplished client-side by passing a list of approved_clients into the login status API call:
 
 ```js
-let credential = await navigator.credentials.get({
-  identity: {providers: {origin: "https://login.idp.net"}}});
-let dataFromTheIDP = credential.token;
-if (credential) {
-  let idpConfigSelected = credential.origin;
-} else {
-  // User did not select an account.
-}
-
+navigator.login.setStatus("logged-in", {
+	accounts: [{
+         // ... other fields
+         approved_clients: ["https://rp.example"],
+         // ... other fields
+      }]
+});
 ```
 
-To use cross site cookies, if the credential can be silently accessed by the RP, then a browser may decide there is no additional privacy loss associated with access to unpartitioned state and choose to automatically grant access to Storage Access requests, as [proposed already for FedCM](https://github.com/explainers-by-googlers/storage-access-for-fedcm).
+### Handling Selective Disclosure with the Field Selector
+
+An RP can also select which attributes of the user profile they would like the IdP or user agent to disclose. Currently, the following values are allowed: name, email, picture, phoneNumber and empty list. This aligns with [the current custom requests proposal](https://github.com/w3c-fedid/custom-requests/issues/4).
+
+This would also work client-side only, with the returned credential object only containing the specific attributes that were requested if they were stored in the setStatus call.
```

Comment:
```suggestion
An RP can also select which attributes of the user profile they would like the IdP or user agent to disclose. Currently, the following values are allowed: `name`, `email`, `picture`, `phoneNumber`, and empty list. This aligns with [the current custom requests proposal](https://github.com/w3c-fedid/custom-requests/issues/4).

This would also work client-side only, with the returned credential object only containing the specific attributes that were requested if they were stored in the `setStatus` call.
```